local-config/nats/nats-server.conf

```diff
@@ -1,3 +1,9 @@
+tls: {
```

Comment:
Do these settings allow us to omit some of the CLI flags on the docker image?

Reply:
yes, I removed 3 flags from docker-compose